### ISSUE TITLE
the eslint of 6.6.0 occur errors

### ISSUE THREAD

**Issue by kobe990** [eslint/eslint#12517]
**Tell us about your environment**

* **ESLint Version:** 6.6.0
* **Node Version:** 8.9.1
* **npm Version:** 6.10.1

**What parser (default, Babel-ESLint, etc.) are you using?**

**Please show your full configuration:**
** This the webpack config
```js
rules: [
      {
        test: /\.(js|vue)$/,
        loader: 'eslint-loader',
        enforce: 'pre',
        include: [resolve('src'), resolve('test')],
        options: {
          formatter: require('eslint-friendly-formatter'),
          quiet: false,
        }
      },
      {
        test: /\.vue$/,
        loader: 'vue-loader',
      },
      {
        test: /\.js$/,
        loader: 'babel-loader',
        include: [resolve('src'), resolve('test')]
      }]
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**
I just run the command: `npm run dev`,

**What did you expect to happen?**
no errors happen.

**What actually happened? Please include the actual, raw output from ESLint.**
![image](https://user-images.githubusercontent.com/7878020/68021350-21b49c00-fcdc-11e9-9ed9-9046dd297dfe.png)

the errors on the picture happened. When I downgrade the version of the eslint. the errors disappear. It seems that the 6.6.0 version of the eslint has this problem

**Comment by mysticatea** [comment]
Thank you for your report.

This is a duplicate of https://github.com/eslint/eslint/issues/12493.

ESLint doesn't support Node.js 8.9.1 since ESLint 6.0.0. Please use `>=8.10.0`. See also [Migrating to 6.0.0](https://eslint.org/docs/user-guide/migrating-to-6.0.0#drop-node-6).
